openmc/mgxs/mgxs.py

```diff
@@ -1625,14 +1773,12 @@ def get_pandas_dataframe(self, groups='all', nuclides='all',
 
         # If the user requested a specific set of nuclides
         elif self.by_nuclide and nuclides != 'all':
-            query_nuclides = nuclides
```

Comment:
Did you discover that `query_nuclides` is unnecessary? I recall putting it in here for a reason, but don't remember what it was.